README.md

```diff
@@ -52,6 +52,26 @@ make VERSION=0.3.0 push
 make clean
 ```
 
+On Windows, using the tools described below, use the command:
+
+"C:\Program Files\git\bin\bash" -c "make"
```

Comment:
These work for me in Command Prompt, but not in Powershell. In Powershell I get the following error:

```
At line:1 char:33
+ "C:\Program Files\git\bin\bash" -c "make"
+                                 ~~
Unexpected token '-c' in expression or statement.
At line:1 char:36
+ "C:\Program Files\git\bin\bash" -c "make"
+                                    ~~~~~~
Unexpected token '"make"' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken
```
If getting this to run in powershell is low-effort, it might be nice to have. Otherwise I'm not too concerned, and we just need to specify to use Command Prompt and not PowerShell.